test/cucumber/ClassStats.feature

```diff
@@ -0,0 +1,21 @@
+#Jorge Vin�cius Diniz e Lima - jvdl
+Feature: Show statistics about the class
+	As a teacher
+	I want to see the percentage of students who passed, who is participating at the forum, and accessing the website
+	So that I can know if I am teaching the class right
+
+Scenario: Show percentage of students class
+	Given all students received their evaluations
+	And I am at the Assistant Teacher's statistics page
+	When I pres the "Show Statistics"
+	Then I should see the percentage of students who achieved MANA
+	And I should see the percentage of students who achieved MPA
+	And I should see the percentage of students who achieved MA
```

Comment:
could be: Then I should see the percentage of students who achieved MANA, MPA and MA